### PR TITLE
Create ubuntu18.04-minimal

### PR DESCRIPTION
**Pull request by nadroj94**
Tested with ubuntu18.04Python to be used as the base minimal ubuntu 18.04 for Databricks Runtime 7.1 and 7.2
[env.txt](https://github.com/databricks/containers/files/5199057/env.txt)
 - Example env.txt will need to be included in build replacing the env.yml

### CODE REVIEW

File: experimental/ubuntu/ubuntu18.04-minimal/ubuntu18.04-minimal [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+RUN apt-get update \
+  && apt-get install --yes \
+    openjdk-8-jdk \
+    iproute2 \
+    bash \
+    sudo \
+    coreutils \
+    procps \
+    strace \
```
Comment (evanye) [inline experimental/ubuntu/ubuntu18.04-minimal/ubuntu18.04-minimal:11]: why do we need strace? this is different from before.

**Review by evanye — COMMENTED** [review]
thanks for doing this! can you also help add:

- A readme similar to https://github.com/databricks/containers/blob/master/ubuntu/minimal/README.md that highlights what this is for (e.g. DBR 7+)
- Update the current ubuntu/minimal readme to mention this is for DBR 6-

File: experimental/ubuntu/ubuntu18.04-minimal/ubuntu18.04-minimal (hunk 1 above)
Comment (nadroj94) [reply experimental/ubuntu/ubuntu18.04-minimal/ubuntu18.04-minimal:11]: Strace is needed for the parameter that can be set to activate strace: spark.databricks.daemon.driver.python.withStrace

Comment (nadroj94) [reply experimental/ubuntu/ubuntu18.04-minimal/ubuntu18.04-minimal:11]: And README.md created for 7.x

Comment (evanye) [reply experimental/ubuntu/ubuntu18.04-minimal/ubuntu18.04-minimal:11]: It seems like `withStrace` is false by default. I'd leave this commented out with a note saying that strace should be installed for users who are trying to set `withStrace` on their clusters

Comment (evanye) [reply experimental/ubuntu/ubuntu18.04-minimal/ubuntu18.04-minimal:11]: where's the README?

Comment (nadroj94) [reply experimental/ubuntu/ubuntu18.04-minimal/ubuntu18.04-minimal:11]: It is included in a separate pull request.

Comment (evanye) [reply experimental/ubuntu/ubuntu18.04-minimal/ubuntu18.04-minimal:11]: @nadroj94 please leave a comment for strace and we can merge this. thanks!

Comment (nadroj94) [inline experimental/ubuntu/ubuntu18.04-minimal/ubuntu18.04-minimal:11]: strace should be installed for users who are trying to set withStrace on their clusters

**Review by evanye — APPROVED** [review]

**Pull request merged**